core-bundle/tests/EventListener/SearchIndexListenerTest.php

```diff
@@ -52,15 +52,15 @@ public function getRequestResponse(): \Generator
     {
         yield 'Should index because the response was successful and contains ld+json information' => [
             Request::create('/foobar'),
-            new Response('<html><body><script type="application/ld+json">{"@context":"https:\/\/contao.org\/","@type":"PageMetaData","pageId":2,"noSearch":false,"protected":false,"groups":[],"fePreview":false}</script></body></html>'),
+            new Response('<html><body><script type="application/ld+json">{"@context":"https:\/\/contao.org\/","@type":"PageMetadata","pageId":2,"noSearch":false,"protected":false,"groups":[],"fePreview":false}</script></body></html>'),
```

Comment:
Unrelated to this pull request, but this should probably be changed to `RegularPage` since https://github.com/contao/contao/commit/b59bd536070d96decc349b7eaec2142be0687f4e

Reply:
I reverted the respective parts in the `SearchIndexListenerTest` and `DocumentTest` for this PR in 2692f080a999af1a02f3f53fc21fba18ebc9ae55.

Reply:
Pull request for the type changes: https://github.com/contao/contao/pull/2055